deploy/mlflow-triton-plugin/README.md

```diff
@@ -66,7 +66,7 @@ OUTPUT1 is the element-wise subtraction of INPUT0 and INPUT1.
 ### Start Triton Inference Server in EXPLICIT mode
 
 The MLflow Triton plugin must work with a running Triton server, see
-[documentation](https://github.com/triton-inference-server/server/blob/main/docs/getting_started/quickstart.md)
+[documentation](https://github.com/triton-inference-server/server/blob/r22.10/docs/getting_started/quickstart.md)
```

Comment:
Same here. Are we going to point to latest release instead of dev branch on dev branch?

Reply:
@tanmayv25 - ack will fix it